agent.go

```diff
@@ -495,7 +500,7 @@ func (s *sandbox) listenToUdevEvents() {
 		})
 
 		// Check if device hotplug event results in a device node being created.
-		if uEv.DevName != "" && uEv.Action == "add" && strings.HasPrefix(uEv.DevPath, rootBusPath) {
+		if uEv.DevName != "" && strings.HasPrefix(uEv.DevPath, rootBusPath) {
```

Comment:
Same thing here, this should go to a separate commit.

Reply:
Same here. It **IS** a single commit.